src/lib.rs

```diff
@@ -350,10 +359,9 @@ fn locate_template_file(
     }
 }
 
-fn resolve_project_dir(name: &ProjectName, args: &Args) -> Result<PathBuf> {
+fn resolve_project_dir(base_dir: &Path, name: &ProjectName, args: &Args) -> Result<PathBuf> {
```

Comment:
FYI: the parameter `base_dir` will come with #625 as a `destination` parameter as part of `Args`

Reply:
I've rebased my stuff and made changes to respect the new `--destination` option, thus this will not be an issue.